### ISSUE TITLE
Venezuela VATs

### ISSUE THREAD

**Issue by kkaiser** [arthurdejong/python-stdnum#97]
Hello,

Do you think it would be possible to at VAT numbers of Venezuela. The validation in JS can be seen here: https://github.com/formvalidation/formvalidation/blob/master/src/js/validator/vat.js#L1378 and tested here: https://formvalidation.io/guide/validators/vat/venezuelan-vat-number/

At the moment I'm just using something similar to this:
```python
def validate_ve_vat_number(number):
    def compact(number):
        number = clean(number, ' -').upper().strip()
        if number.startswith('VE'):
            number = number[2:]
        return number

    number = compact(number)
    if number[0] not in 'VEJPG': #or not isdigits(number):
        raise InvalidFormat()
    if len(number) != 10:
        raise InvalidLength()
    company_type = {
        'V': 4,
        'E': 8,
        'J': 12,
        'P': 16,
        'G': 20
    }
    base_sum = company_type[number[0]]
    weights = (3, 2, 7, 6, 5, 4, 3, 2)
    base_sum += sum(weights[i] * int(n)
                   for i, n in enumerate(number[1:9]))
    base_sum = 11 - base_sum % 11
    if base_sum == 11 or base_sum == 10:
        base_sum = 0
    if base_sum != int(number[9]):
        raise InvalidChecksum()
    return number
```

And some additional information for format is here: https://en.wikipedia.org/wiki/VAT_identification_number#VAT_numbers_of_Latin_American_countries

**Comment by kkaiser** [comment]
Btw there is a really nice documentation for many countries here: https://wiki.scn.sap.com/wiki/display/CRM/Country+Tax+Category+check

**Comment by arthurdejong** [comment]
Thanks for pointing it out and providing the useful links and resources. I have implemented the format in 4ac84c5.

I have quite a list of candidate numbers I could implement ;) but suggestions are always welcome.
